bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/common/DefaultServerController.java

```diff
@@ -41,6 +41,7 @@ public void configure(final Context ctx) {
       try {
          if (updateServerConfiguration(newConfiguration)) {
             modelRepository.initialize();
+            // ContextResponse.success(ctx);
```

Comment:
Seems like an uncessary line to me

Reply:
Oh my bad, this was the whole point of the fix, I added this line again in the next commit :+1: